notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2224 @@
+{
```

Comment:
<blockquote>Now load it from the Cloud Storage bucket, and then you can do some things, like evaluate the model, and do a prediction.</blockquote><p>Update: Now load it from the Cloud Storage bucket, and then you can perform tasks such as model evaluation and make prediction requests.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>done</p>